varcode/variant_annotator.py

```diff
@@ -11,7 +12,7 @@ class VariantAnnotator(object):
     def __init__(self, ensembl_release):
         self.ensembl = pyensembl.EnsemblRelease(ensembl_release)
 
-    def describe_variant(self, variant):
+    def describe_variant(self, variant, raise_on_error=True):
```

Comment:
Should `raise_on_error` be in the `VariantAnnotator` constructor instead of peppered across different methods?